easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.8.2-no-cpp-patch.eb

```diff
@@ -1,11 +1,16 @@
 name = 'ncurses'
 version = '5.9'
+versionsuffix = '-no-cpp-patch'
 
 homepage = 'http://www.gnu.org/software/ncurses/'
 description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
  and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
  function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
 
+import platform
+
+patches = ['configure_darwin.patch']
```

Comment:
include the software name/version in the patch: `ncurses-5.9_configure_darwin.patch`